implementationGuide/Mappings.md

```diff
@@ -36,14 +39,16 @@ In FHIR, the contact information is included in the *Organization* resource, whi
 |address | contact:*contact*.address.line |
 |zip | contact:*contact*.address.postalCode |
 |city | contact:*contact*.address.city |
-|country | contact:*contact*.address.county |
-
-### Collection
-### Network
+|country | contact:*contact*.address.country |
 
+[1]:https://www.hl7.org/fhir/organization.html
```

Comment:
Can you please put this at the end of the file. Then the links will be easier to find. Especially the next free link number.

Reply:
Good point!